packages/daimo-contract/src/tokenMetadata.ts

```diff
@@ -0,0 +1,9 @@
+// Generated by script/fetchTokenMetadata.ts
+
+export const tokenMetadata = {
+  address: "0x1B85deDe8178E18CdE599B4C9d913534553C3dBf",
+  decimals: 6,
+  symbol: "USDC",
+  name: "testUSDC",
+  totalSupply: "1000000000000",
```

Comment:
tbh this is a tiny amount of metadata

for a huge decision (which coin Daimo runs on)

idk if codegen is necessary for that

(not a big deal though; checking it in to `daimo-contract` so we can import everywhere else lgreattm)

Reply:
yeah, good point!